### ISSUE TITLE
Accessibility issue #1 <h1> should be <h3>

### ISSUE THREAD

**Issue by sfrucht** [mitodl/sga-lti#57]
A heading is out of order. In Student view, the unit title (in the case below "Assignment") is labeled as a <h1>, where it should be an <h3> as it is in Admin view. 
<img width="846" alt="screen shot 2016-08-02 at 10 09 29 am" src="https://cloud.githubusercontent.com/assets/899424/17331525/515deba4-5899-11e6-8663-09f072e47cb2.png">
